app/dturn/__tests__/add-retrieve-1-statement.js

```diff
@@ -0,0 +1,37 @@
+const { Mongo } = require('@enciv/mongo-collections')
+const { initDiscussion, insertStatementId, getStatementIds, Discussions } = require('../dturn')
+
+const DISCUSSION_ID = 'testDiscussion'
+const USER_ID = 'user1'
+const STATEMENT_ID = 'statement1'
+const OPTIONS = {
+  group_size: 10,
+  gmajority: 0.5,
+  max_rounds: 10,
+  min_shown_count: 6,
+  min_rank: 3,
+  updateUInfo: () => {},
+  getAllUInfo: async () => [],
+}
+
+beforeAll(async () => {
```

Comment:
It works great, but the Mongo db isn't being users and doesn't need to be initialized - and someone in the future might be confused be that. So can the the require@enciv/mongo..., the Mongo.connect and Mongo.disconnect be removed form this and some of the other files.

Reply:
Please remove line 1, 18, 22-25. It's not needed here.  Of if I'm wrong let me know. Thanks.